js/main.js

```diff
@@ -141,7 +141,7 @@ var nav = new mapboxgl.NavigationControl({
   showZoom: true
 });
 
-mapRegular.addControl(nav, "top-left");
+mapRegular.addControl(nav, "top-right");
```

Comment:
I think this is the better option, if for no other reason than it seems standard, but I am curious if you had it on the left initially for a particular reason?

Reply:
I don't think I had a specific reason for it being in the top left initially. On Azavea and external mapping projects, I've see the zoom buttons in all corners, and I think any location is fine.

The reason I'm moving it to the top right is to move it away from the "about" button in the sidebar, so there aren't as many distracting buttons in that area. I think this is a bit more clear.

Reply:
That makes sense. And thanks for the explanation.